blink the power-off switch in red

tophf opened the pull request:
It blinks only when the styles area is hovered, which probably means the user cluelessly tries to enable it.

![2022-03-10_22-28-31](https://user-images.githubusercontent.com/1310400/157739969-edb74c00-c5dc-4a31-b13b-7136e0ff10ab.gif)

tophf commented:
Also added the switch in the manager because many users might open it without ever opening the popup.

![image](https://user-images.githubusercontent.com/1310400/157741709-63f3f5b9-0a67-492c-90dd-103e481cc656.png)

narcolepticinsomniac commented:
Solid idea. Might also blink both on page load, for a few seconds. Checkbox at the bottom in manager might be better at the top? It's decent as-is, just a couple suggestions from my first impression.

tophf commented:
Blinking seems weird after all. I've switched to showing a red background on the switch:

![image](https://user-images.githubusercontent.com/1310400/158304307-0fcca28a-739b-4966-a225-2a81fb1838eb.png) ![image](https://user-images.githubusercontent.com/1310400/158305234-05ecd9b6-cf69-46fa-b157-a42cd6808366.png)

The checkbox was at the top in the manager at first, but then I thought it's not that frequently used to justify its weird lonely presence. I've changed this PR to show the checkbox only when it's enabled - either prior to opening the manager or elsewhere (popup/hotkey) while it's shown.

![image](https://user-images.githubusercontent.com/1310400/158306887-2c50baf4-7e97-4033-97bc-90b26afa2f09.png)

After it's shown it will stay visible even if unchecked:

![image](https://user-images.githubusercontent.com/1310400/158306927-88417ac3-72e5-4e84-9309-07487052a560.png)

narcolepticinsomniac commented:
Blinking was fine, IMO. I don't particularly like the red bg, and this current implementation seems pretty buggy. Not so bad in the popup, but it looks weird in the manager.

Also causing horizontal overflow in the header for some strange reason, and checkbox hover styling is mucked up in the light theme. And why would the dark theme require a lighter shade of red for a bg?

Even if the bugs are fixed, I preferred it the way it was, or maybe some different indication entirely.

tophf commented:
Since most users don't have a problem understanding the current UI, there should be no need to blink or otherwise over-emphasize the label, so I've reverted blinking and red background.

Instead, the switch will be shown in all pages (popup, manager, editor). No changes in the popup. The manager/editor show the switch on page load only if it was active, also if it was enabled elsewhere while the page is open. Once it's shown the switch won't hide anymore.

The new text for the active state:

page | UI
-|-
manager | ![image](https://user-images.githubusercontent.com/1310400/160601287-5aeb78e6-6bde-4c99-a054-e0a456cd3dcf.png)
popup | ![image](https://user-images.githubusercontent.com/1310400/160601314-9af02812-a1da-4445-b5e4-3d42535a1ea0.png)
editor | ![image](https://user-images.githubusercontent.com/1310400/160601353-c8655788-e04a-4a7c-8a9f-be5c5309b17f.png)
